.github/renovate.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I think we'd want to skip the ASP.NET related dependencies in Google.Apis.Auth.Mvc, Google.Apis.Auth.AspNetCore and Google.Apis.Auth.AspNetCore3, but I can do that on Monday when the PRs start coming on, so we fine tune it to what renovate tries to do.

Reply:
Yup, that's exactly the strategy I'm thinking of :)